python/cuml/cuml/accel/_wrappers/sklearn/manifold.py

```diff
@@ -17,7 +17,11 @@
 import cuml.manifold
 from cuml.accel.estimator_proxy import ProxyBase
 
-__all__ = ("TSNE",)
+__all__ = ("SpectralEmbedding", "TSNE")
+
+
+class SpectralEmbedding(ProxyBase):
+    _gpu_class = cuml.manifold.SpectralEmbedding
```

Comment:
For estimators where we don't support all fitted attributes, we want to list all unsupported attributes in the `_not_implemented_attributes` set like so:

```suggestion
    _gpu_class = cuml.manifold.SpectralEmbedding
    _not_implemented_attributes = frozenset(("affinity_matrix_",))
```